public/urls.json

```diff
@@ -8,6 +8,12 @@
   "bpo-log-in": "https://portal.edd.ca.gov/WebApp/Login",
   "bpo-log-out": "https://portal.edd.ca.gov/WebApp/Logout",
   "uio-desktop-help-new-claim": "https://uio.edd.ca.gov/UIO/Pages/Public/help/index.htm#t=en-US/Public/NewClaim/UIOnlineNewClaimLandingPage.htm",
-  "uio-desktop-home-url": "https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx",
-  "uio-mobile-home-url": "https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
+  "uio-desktop-home-url": "https://uio.edd.ca.gov/UIO/Pages/ExternalUser/ClaimantAccountManagement/UIOnlineHome.aspx",
+  "uio-mobile-home-url": "https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx",
```

Comment:
Note: I noticed these links differ from each other (desktop vs mobile) so I went back to look through my emails. I can confirm that UIO confirmed via email on Sept 29 and Sept 30 (thread subject "RE: Update CST Test environment for global header demo") that this is the correct behavior. We implemented it as part of #505